src/entities/CheckScenes/task/taskRunnerSqs.ts

```diff
@@ -43,6 +48,50 @@ const placesAttributes: Array<keyof PlaceAttributes> = [
   "textsearch",
 ]
 
+const ACCESS_KEY = env("AWS_ACCESS_KEY")
+const ACCESS_SECRET = env("AWS_ACCESS_SECRET")
+const BUCKET_HOSTNAME = env("BUCKET_HOSTNAME")
+const BUCKET_NAME = env("AWS_BUCKET_NAME", "")
+
+async function updateWorldManifest() {
```

Comment:
the name `updateWorldManifest ` is a bit tricky, it's about world, or a genesis city or both?

Reply:
There is no ubiquitous language yet for this but some teammates from E@ are referring to this manifest JSON as `World Manifest`.  Anyways, I've renamed it to `Genesis City Manifest`.